src/Chapter3.hs

```diff
@@ -462,6 +474,8 @@ Create a simple enumeration for the meal types (e.g. breakfast). The one who
 comes up with the most number of names wins the challenge. Use your creativity!
 -}
 
+data Meal = Breakfast Int | Lunch Int | Dinner Int
```

Comment:
Yummy 😋 
What does `Int` mean in constructors? 🙂

Reply:
Calories lol